### DISCUSSION TITLE
Task data

### DISCUSSION THREAD

**Discussion by expressmoney** [viewflow/viewflow#417] [General]
Hello,
How to add any set of models to task data (right block)?
![viewflow any data](https://github.com/viewflow/viewflow/assets/139622411/80dea2fc-420f-472e-abe1-0ccea4cce6be)

**Comment by kmmbvnr** [comment]
Thank you for the question. I've added section to the documentation about templates overriding

https://docs.viewflow.io/workflow/templates.html#process-data

Basically you could create  {app_label}/{flow_label}/process_data.html (see the shipment demo for the reference)
